### PR TITLE
Doc: Fix broken links and typo in  modules-structure.md

### PR DESCRIPTION
**Pull request by BourgoisMickael**
*Description of changes:* documentation:

- Broken links
- Typo

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by mmaeng — APPROVED** [review]

**Comment by mmaeng** [comment]
Closing and reopening to kick the tests

File: doc_source/modules-structure.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ A module consists of two main pieces:
 ## Creating the module template fragment<a name="modules-template-fragment"></a>
 
 The starting point for developing a module is the template fragment\. The template fragment is a file that contains the information that defines the resources for CloudFormation to provision during stack operations, including:
-+ A `[Resources](https://docs.aws.amazon.com/)` section that defines the resources to be provisioned\.
++ A `[Resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html)` section that defines the resources to be provisioned\.
```
Comment (cmgorton) [inline doc_source/modules-structure.md:10]: We should remove the back-ticks around the Resources link. The back-ticks prevent it from being linkable.

**Review by cmgorton — CHANGES_REQUESTED** [review]

**Review by cmgorton — APPROVED** [review]

File: doc_source/modules-structure.md (hunk 1 above)
Comment (BourgoisMickael) [reply doc_source/modules-structure.md]: Thanks for reviewing, I've removed the back-ticks

Comment (cmgorton) [reply doc_source/modules-structure.md]: Thank you! Running the test now. When they pass we should be able to merge this PR.

**Pull request merged**